ts/nni_manager/training_service/kubernetes/kubernetesApiClient.ts

```diff
@@ -158,6 +159,14 @@ abstract class KubernetesCRDClient {
 
     protected abstract get operator(): any;
 
+    public set setNamespace(namespace: string) {
```

Comment:
Seem it's not needed and this is not how setter works. It's like following code in Python:
```python
@property
def getNamespace(self):
    return self.namespace

@setNamespace.setter
def setNamespace(self, namespace):
    self.namespace = namespace
```

Reply:
just follow code style in `GeneralK8sClient `...

Reply:
If you want to use property, it should be:
```typescript
public get namespace(): string {
    return this.privateNamespace;
}
public set namespace(ns: string): void {
    this.privateNamespace = ns;
}
```
Or if you want to use getter method, it should be:
```typescript
public getNamespace(): string {
    return this.namespace;
}
public setNamespace(namespace: string): void {
    this.namespace = namespace;
}
```

Reply:
I suggest to simply make the field public...